actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -614,8 +614,6 @@ private void validateUserTypeForUpdate(Map<String, Object> userMap, boolean isCu
         ProjectCommonException.throwClientErrorException(
```

Comment:
We do not need this teacher validation any more

Reply:
ok sure.. Removing the validation